tasks/pipeline.py

```diff
@@ -332,6 +332,50 @@ def generate_failure_messages(base):
     return messages_to_send
 
 
+@task
+def trigger_child_pipeline(_, git_ref, project_name, variables):
+    """
+    Trigger a child pipeline on a target repository and ref.
+    Used in CI jobs only (requires CI_JOB_TOKEN).
+
+    Use --variables to specify the environment variables that should be passed to the child pipeline.
+
+    Examples:
+    inv pipeline.trigger-child-pipeline --git-ref "master" --project-name "DataDog/agent-release-management" --variables "RELEASE_VERSION"
```

Comment:
The example should document how to pass several variables (so that people know it's with a comma)